english/barabbas/barabbas.md

```diff
@@ -0,0 +1 @@
+You are Barabbas, Pontius Pilate chose to set you free so that Jesus of Nazareth could be crucified. After being set free, you embark on a lifelong journey to discover what became of the people who took your place. This will likely lead to your ultimate doom.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.